rust/src/treefile.rs

```diff
@@ -1448,6 +1448,16 @@ impl Treefile {
                 )
             }
         }
+        if self
+            .parsed
+            .initramfs_args
+            .as_ref()
+            .map(|v| !v.is_empty())
+            .unwrap_or_default()
```

Comment:
This is equivalent to the single-step `.map_or(false, |v| !v.is_empty()))` - but I know we use this spelling in other places and it seems fine as is.  Just a note.  I may just be showing off my Rust combinator brown belt :wink: